Storage/SQL/BrowserTable.swift

```diff
@@ -100,7 +104,7 @@ private let log = Logger.syncLogger
  * We rely on SQLiteHistory having initialized the favicon table first.
  */
 public class BrowserTable: Table {
-    static let DefaultVersion = 17    // Bug 1298918.
+    static let DefaultVersion = 19    // Bug 1303731.
```

Comment:
You can bump it up to 18 and when I go back to the metadata storage stuff I'll set it to 19.